content/momentum/4/production-config.md

```diff
@@ -1,285 +1,159 @@
+| Message Spools* | `/var/spool/ecelerity` | 2 x 300 GiB in RAID1 |
+| OS<br>App Binaries<br>Logs<br>Platform DB<br>Analytics DB | `/` (root)<br>`/opt/msys`<br>`/var/log/ecelerity`<br>`/var/db/cassandra`<br>`/var/db/vertica` | 2 x 300 GiB in RAID1 |
 
-*   Analytics DB - /var/db/vertica
+(*) _This array should be dedicated to the spools._
 
- |
+### <a name="production.config.standard.cluster"></a> Enterprise Standard
 
-### <a name="production.config.standard.cluster"></a> Enterprise Standard Cluster
+The Enterprise Standard configuration consists of three nodes running all MTA roles with the resources specified below.
 
-The Enterprise Standard Configuration consists of three nodes running all roles with the resources specified below. The system supports the following performance ratings.
+#### <a name="production.config.standard.cluster.perf"></a> Performance Ratings
 
-<a name="enterprise_standard_performance_ratings"></a> 
-
-
-| Node Capacity | 
-
-Cluster Capacity
-
-(2 Nodes Operational)
-
- | 
-
-Peak Cluster Capacity
-
-(3 Nodes Operational)
-
- |
+| MTA Node Capacity | Cluster Capacity<br><em>(2 Nodes Operational)</em> | Peak Cluster Capacity<br><em>(3 Nodes Operational)</em> |
 | --- | --- | --- |
-| 1 M Msg/hr | 2 M Msg/hr | 3 M Msg/hr |
-
-<a name="enterprise_standard_cluster_hw_reqs"></a> 
+| 3 M msgs/hr | 6 M msgs/hr | 9 M msgs/hr |
 
+#### <a name="production.config.standard.cluster.hw_specs"></a> Hardware Specifications
 
-| Resource | Minimum Specification |
+| Resource | Specification |
 | --- | --- |
-| CPU | 16 x 2.5 GHz Cores (Min Speed) |
-| Memory | 64 GB RAM |
-| Network Interface | 1 GB NIC |
+| CPU Cores | 16 |
+| CPU Speed | 3.2 GHz (min. 2.5 GHz) |
+| Memory | 64 GiB (min. 32 GiB) RAM |
+| Network Interface | 1 Gbps NIC |
 
-<a name="enterprise_standard_cluster_storage_array"></a> 
+#### <a name="production.config.standard.cluster.storage_array"></a> Storage Configuration
 
-
-| Array | Configuration | Mount Points and Notes |
+| Array | Mount Points | Configuration |
 | --- | --- | --- |
-| All Storage | 8 x 300 GB 15k RPM HDD |   |
-| Message Spools | 4 x 300 GB in RAID10 | 
-
-/var/spool/ecelerity
-
-Note: This array should be dedicated to the spools.
-
- |
-| OS, App Binaries, Logs, Platform DB | 2 x 300 GB in RAID1 | 
-
-*   OS - / (root)
-
-*   Logs - /var/log/ecelerity
-
-*   App Binaries - /opt./msys
-
-*   Platform DB - /var/db/cassandra
-
- |
-| Analytics DB | 2 x 300 GB in RAID1 | 
-
-Analytics DB - /var/db/vertica
+| All Storage |   | 8 x 300 GiB 15k RPM HDD |
+| Message Spools* | `/var/spool/ecelerity` | 4 x 300 GiB in RAID10 |
+| OS<br>App Binaries<br>Logs<br>Platform DB | `/` (root)<br>`/opt/msys`<br>`/var/log/ecelerity`<br>`/var/db/cassandra` | 2 x 300 GiB in RAID1 |
+| Analytics DB* | `/var/db/vertica` | 2 x 300 GiB in RAID1 |
 
-Note: This array should be dedicated to the Analytics DB.
+(*) _These arrays should be dedicated._
 
- |
+### <a name="production.config.plus.cluster"></a> Enterprise Plus
```

Comment:
Should we call out any Momentum-specific configuration options that are used for `Enterprise Plus` or other configurations? e.g., supercharger?